client-sdk/ts-web/core/src/beacon.ts

```diff
@@ -1,3 +1,24 @@
+/**
+ * MethodPVSSCommit is the method name for a PVSS commitment.
+ */
+export const METHOD_PVSS_COMMIT = 'beacon.PVSSCommit';
+/**
+ * MethodPVSSReveal is the method name for a PVSS reveal.
+ */
+export const METHOD_PVSS_REVEAL = 'beacon.PVSSReveal';
+/**
+ * MethodSetEpoch is the method name for setting epochs.
+ */
+export const METHOD_SET_EPOCH = '000_beacon.SetEpoch';
```

Comment:
Not sure if these need to be exposed in the client SDK as the first two can only be called by a validator and the last one can only be used in debug mode.

Reply:
I'm fine with exposing methods used in debug mode.